manager/integration/tests/test_recurring_job.py

```diff
@@ -55,6 +64,45 @@ def wait_until_begin_of_an_even_minute():
         time.sleep(1)
 
 
+def wait_for_recurring_backup_to_start(client, core_api, volume_name, n, progress=50):  # NOQA
+    job_pod_name = volume_name + '-backup-c'
+    snapshot_name = ''
+    snapshots = []
+    check_pod_existence(core_api, job_pod_name, namespace=LONGHORN_NAMESPACE)
+
+    # Find the snapshot which is being backed up
+    for _ in range(RETRY_BACKUP_COUNTS):
+        volume = client.by_id_volume(volume_name)
+        try:
+            snapshots = volume.snapshotList()
+        except (AttributeError, ApiException):
+            time.sleep(RETRY_BACKUP_INTERVAL)
+            continue
+        if len(snapshots) == n + 1:
+            for snapshot in snapshots:
+                if snapshot.children['volume-head']:
+                    snapshot_name = snapshot.name
+            break
+    assert snapshots is not None
```

Comment:
Why do we need to find the snapshot in this function?
I thought that we just need to check `v.backupStatus` and wait until it is. >= progress %
There is only 1 backup cronjob running so it is not possible that the backup process causes by a different job.

Reply:
`wait_for_backup_completion` will require the snapshot name as a parameter, so it's efficient to get it while we're checking backup operation is started.